src/layer.rs

```diff
@@ -250,7 +250,11 @@ impl ApmLayer {
         };
 
         ApmLayer {
-            client: ApmClient::new(config.apm_address, config.authorization),
+            client: ApmClient::new(
+                config.apm_address,
+                config.authorization,
+                config.allow_invalid_certs.unwrap_or(false),
```

Comment:
Instead of unwrapping to a default value, it's better to simply set the default and not use Option at all.